Applications/Games/XIII/GOG/script.js

```diff
@@ -0,0 +1,20 @@
+include(["engines", "wine", "quick_script", "gog_script"]);
+
+var installerImplementation = {
+    run: function () {
+        new GogScript()
+            .name("XIII")
+            .editor("Ubisoft")
+            .applicationHomepage("")
+            .author("Quentin PÂRIS")
+            .gogSetupFileName(["xiii/en1installer0", "xiii/en1installer1", "xiii/en1installer2"])
+            .category("Games")
+            .wineVersion(LATEST_STABLE_VERSION)
```

Comment:
`LATEST_STABLE_VERSION` and `upstream` is default. No need to set it explicitly.